src/GovernorCountingFractional.sol

```diff
@@ -241,4 +245,79 @@ abstract contract GovernorCountingFractional is Governor {
             abstainVotes := shr(128, mload(add(voteData, 0x40)))
         }
     }
+
+
+    /**
+     * @dev See {IGovernor-castVoteWithReasonAndParamsBySig}.
+     * TODO add new docs
+     * the final 128bits of the params should contain the nonce.
+     */
+    function castVoteWithReasonAndParamsBySig(
```

Comment:
Completely unsolicited feedback, I prefer this impl because it sticks to OZ Gov standard 👍

Reply:
Thanks again for the quick work on this issue!

Reply:
Of course, thanks for letting us know about it. Appreciate the feedback!